code/game/objects/items/weapons/storage/trays.dm

```diff
@@ -16,9 +16,8 @@
 	use_to_pickup = 1
 	allow_quick_gather = 1
 	use_sound = null
-	material = /decl/material/solid/cardboard
+	material = /decl/material/solid/metal/aluminium
```

Comment:
`/obj/item/storage/tray/metal/aluminium` already exists, are we just removing cardboard entirely? I don't see why cardboard food trays would need to be cut or replaced.
![TR-SC-UF_five_compartment_tray_330](https://github.com/NebulaSS13/Nebula/assets/110272328/d34b1177-a965-44c1-b390-9065e88a0d1e)
^ This is made of paper.